lib/node_modules/@stdlib/math/base/special/ldexp/README.md

```diff
@@ -118,6 +118,98 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/ldexp.h"
+```
+
+#### stdlib_base_ldexp( frac, exp )
+
+Multiplies a [double-precision floating-point number][ieee754] by an `integer` power of two; i.e., `x = frac * 2^exp`.
+
+```c
+#include <stdint.h>
+stdlib_base_ldexp( 0.5, 3 ); // => 0.5 * 2^3 = 0.5 * 8
+```
+
+The function accepts the following arguments:
+
+-   **frac**: `[in] double` input value.
+-   **exp**: `[in] int32_t` integer power of two.
+
+```c
+double stdlib_base_ldexp( const double frac, const int32_t exp );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/ldexp.h"
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <math.h>
+#include <inttypes.h>
+int main() {
+    double frac;
+    int32_t exp;
+    double out;
+    int i;
+
+    for ( i = 0; i < 100; i++ ) {
+        frac = (double) rand() / (double) RAND_MAX;
+        exp = (int32_t) rand() - (int32_t) (RAND_MAX/2);
```

Comment:
Exponents can never be this large. My preference would be to, per usual C examples, have a list of double-precision floats, and then, similar to the JS example, use `frexp` to split into fraction and exp and then reconstitute. The result should be values which round trip.

Reply:
When I try to merge changes from `develop` to this PR git shows me an `unrelated commit history` error, and if I try to `git merge develop --allow-unrelated-commit-history` there are quite a lot of conflicts that are rising.

Reply:
Hmm...not sure why you should have any conflicts with the current `develop`.